doc/api/tls.markdown

```diff
@@ -352,16 +356,16 @@ gets high.
 This is a wrapped version of [`net.Socket`][] that does transparent encryption
 of written data and all required TLS negotiation.
 
-This instance implements a duplex [Stream][] interfaces.  It has all the
+This instance implements the duplex [Stream][] interfaces. It has all the
```

Comment:
Nit: wouldn't it be more correct to say it implements the duplex stream _interface_ (singular)?